platform-controller/src/test/java/org/hobbit/controller/docker/ResourceInformationCollectorTest.java

```diff
@@ -7,6 +7,8 @@
 import org.apache.jena.sparql.vocabulary.DOAP;
 import org.apache.jena.vocabulary.RDF;
 import org.apache.jena.vocabulary.RDFS;
+import org.hobbit.controller.containers.ResourceInformationCollector;
+import org.hobbit.controller.containers.docker.ResourceInformationCollectorImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.